modules/byo-vpc-single-az.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * assemblies/byo-vpc.adoc
+
+[id="byo-vpc-single-az_{context}"]
+= BYO VPC with a single availability zone required fields
+
+. Create two subnets as follows:
++
+|===
+| Subnet name | CIDR IP Address
+| `Public`  | `10.0.0.0/17`
+| `Private` | `10.0.128.0/17`
+|===
+
+. Create a `RouteTable` that is associated with the `Public` subnet.
+. Create a routing rule on the RouteTable for `0.0.0.0/0` to the Internet gateway.
+. Create an NAT gateway in the `Public` subnet with an elastic Internet address (EIP).
+. Create a `RouteTable` that is associated with the `Private` subnet.
+. Create a routing rule on the RouteTable for `0.0.0.0/0` to the created NAT gateway.
+
+=== OCM Configuration
+
+.Procedure
+
+. Log in to {cloud-redhat-com}.
```

Comment:
The url of ocm console had been changed to `console.redhat.com`

Reply:
Updated our variables for the correct URL.